### ISSUE TITLE
Old Tika Version Pulling Wrong Width from Image for Metadata

### ISSUE THREAD

**Issue by waqasakramdot** [dotCMS/core#23934]
### Problem Statement

Tika 1.19 returns the same Metadata for two same images but different resolutions.

dotCMS Ticket: https://dotcms.zendesk.com/agent/tickets/109867 
Tika Ticket: https://issues.apache.org/jira/browse/TIKA-2630

### Steps to Reproduce

For the following reproduction steps, I've attached two versions of an image that you can use. They are the same image, but in different sizes, 3600x2400 and 1152x768.

Reproduction steps:

1. Create a new File Asset
2. Upload one version of the image
3. Publish and reopen the file asset
4. Metadata displays correct information
5. Replace the image with the other file
6. Publish and reopen the file asset

Expected results: Metadata is updated to reflect the current live file
Actual results: Metadata still shows the old image's metadata which is inaccurate

First image

![test1](https://user-images.githubusercontent.com/102758033/215476149-8b1310c3-242a-410f-b8e6-00469fc19855.jpg)

Second image

![test2](https://user-images.githubusercontent.com/102758033/215478475-e2ebafac-d289-4a34-bf70-5f68358b829a.jpg)

### Acceptance Criteria

Metadata should correctly be extracted and shown in dotCMS be 

### dotCMS Version

For all dotCMS LTS and Agile versions till the current 23.01

### Proposed Objective

Customer Support

### Proposed Priority

Priority 3 - Average

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

dotCMS Ticket: https://dotcms.zendesk.com/agent/tickets/109867 
Tika Ticket: https://issues.apache.org/jira/browse/TIKA-2630

https://dotcms.slack.com/archives/CSHTYUR7H/p1674828766355419

https://dotcms.slack.com/archives/CSHTYUR7H/p1674831755123549

**Comment by wezell** [comment]
Also:

#[24051](https://github.com/dotCMS/core/issues/24051)

**Comment by fabrizzio-dotCMS** [comment]
Looks good to me. Now the metadata gets recalculated properly showing accurately the respective dimensions

**Comment by fabrizzio-dotCMS** [comment]
On second revision. I can't make site-search pickup contents from large PDF like:
[odisea.pdf](https://github.com/dotCMS/core/files/11391989/odisea.pdf)
[20mil-leguas-viaje-submarino.pdf](https://github.com/dotCMS/core/files/11391990/20mil-leguas-viaje-submarino.pdf)
[viaje-al-centro-de-la-tierra.pdf](https://github.com/dotCMS/core/files/11391991/viaje-al-centro-de-la-tierra.pdf)

**Comment by fabrizzio-dotCMS** [comment]
On Third revision, I finally figured it out
 the portion of the Files that is getting indexed is determined by the following code
```
static long configuredMaxLength () {
    return Config.getLongProperty("META_DATA_MAX_SIZE",
            FileStorageAPI.DEFAULT_META_DATA_MAX_SIZE) * FileStorageAPI.SIZE;
}
```

Meaning that if you want larger pieces of content to be indexed this META_DATA_MAX_SIZE must be increased 
Im passing it to QA

**Comment by josemejias11** [comment]
**Approved QA** - Tested on **23.06_8c0a542e_SNAPSHOT** // Docker // macOS 13.0 // FF v113.0

**Comment by nollymar** [comment]
**Note to QA:** Please make sure blog titles are resolved when a site search index is generated

**Comment by bryanboza** [comment]
Fixed, tested as part of the #24051

**Comment by erickgonzalez** [comment]
It is breaking over 200 tests, so labeling as Excluded from all LTS.
